stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+import '../../dist/list.css';
+import '../../dist/icon.css';
+import '../../dist/object-list.css';
+
+export default {
+    title: 'Components/Object List',
+    parameters: {
+        description: `
+Object list is one of the variant for list items.
+It provides a structure for extending standard list with additional content.
+
+##Usage
+**Use the object List if:**
+
+- You want to display a brief details to user.
+
+**Do not use the object List if:**
+
+- You want to display a detailed information to user.
+
+**The object List comprises:**
+ 
+- Introduction text
+- Avatar
+- Object Identfier
+- Object Number
+- Object Attribute
+- Object Marker
+- Object Status
+
+## Structure
+**Object List consists of the following elements along with standard list:**
+
+  - <code class="docs-code">fd-object-list</code> ul element
+   - <code class="docs-code">fd-object-list\\_\\_item</code> li element
+     - <code class="docs-code">fd-object-list\\_\\_container</code> container to hold child elements
+       - <code class="docs-code">fd-object-list\\_\\_intro</code> introduction text div
+       - <code class="docs-code">fd-object-list\\_\\_header</code> container to hold header elements
+         - <code class="docs-code">fd-object-list\\_\\_header-left</code> container to hold header left elements
+           - <code class="docs-code">fd-object-list\\_\\_object-identifier</code> object identifier element
+         - <code class="docs-code">fd-object-list\\_\\_header-right</code> container to hold header right elements
+           - <code class="docs-code">fd-object-list\\_\\_object-number</code> object number element
+       - <code class="docs-code">fd-object-list\\_\\_content</code> container to hold rows of attributes and status elements
+        - <code class="docs-code">fd-object-list\\_\\_row</code> container to hold attributes and status elements
+         - <code class="docs-code">fd-object-list\\_\\_row-right</code> container to hold right elements of row
+           - <code class="docs-code">fd-object-status</code>, <code class="docs-code">fd-object-marker</code> object status and object status elements
+         - <code class="docs-code">fd-object-list\\_\\_row-left</code> container to hold left elements of row
+           - <code class="docs-code">fd-object-list\\_\\_object-attribute</code> object attribute element
+        
+`,
+        tags: ['f3', 'a11y', 'theme', 'development']
+    }
+};
+
+export const object = () => `
+<h4 id="objectListItemHeader">Object List Item</h4>
+<ul class="fd-list fd-object-list" role="listbox" aria-labelledby="objectListItemHeader">
```

Comment:
Only a selectable list should have a role `listbox`. Non-selectable lists should have the role `list`, which is the default.
```suggestion
<ul class="fd-list fd-object-list" aria-labelledby="objectListItemHeader">
```